specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/operations.json

```diff
@@ -0,0 +1,208 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GraphicalRunbookContent"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OperationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of Automation operations supported by the Automation resource provider."
+        }
+      },
+      "description": "The response model for the list of Automation operations"
+    },
+    "Operation": {
+      "description": "Automation REST API operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.Automation",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Runbooks, Jobs etc.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string"
+            }
+          },
+          "description": "Provider, Resource and Operation values"
+        }
+      }
+    },
+    "GraphicalRunbookContent": {
+      "type": "object",
+      "description": "Graphical Runbook Content",
+      "properties": {
+        "rawContent": {
+          "x-nullable": true,
+          "description": "Raw graphical Runbook content.",
+          "$ref": "#/definitions/RawGraphicalRunbookContent"
+        },
+        "graphRunbookJson": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "Graphical Runbook content as JSON"
+        }
+      }
+    },
+    "RawGraphicalRunbookContent": {
+      "type": "object",
+      "description": "Raw Graphical Runbook content",
+      "properties": {
+        "schemaVersion": {
+          "type": "string",
+          "description": "Schema version of the serializer."
+        },
+        "runbookDefinition": {
+          "type": "string",
+          "description": "Serialized Graphical runbook"
+        },
+        "runbookType": {
+          "type": "string",
+          "description": "Runbook Type",
+          "enum": [
+            "GraphPowerShell",
+            "GraphPowerShellWorkflow"
+          ],
+          "x-ms-enum": {
+            "name": "GraphRunbookType",
```

Comment:
>GraphRunbookType [](http://example.com/codeflow?start=21&length=16)

Will Runbook types always start with only Graph* ? In future, if some other types can be added, consider renaming it to 'RunbookType'. This name matches with that of the property name.

Reply:
Yeah, This is always expected to start with Graph only as this call is specifically for GraphicalRunbooks.

Reply:
sounds good